.ci/build_kubernetes.sh

```diff
@@ -71,13 +71,16 @@ docker image pull ${DOCKER_USERNAME}/microservice-product-search-export:${DOCKER
 # Replace docker images for php-fpm of application and microservices
 yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.containers[0].image ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG}
 yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.initContainers[0].image ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG}
+yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.initContainers[1].image ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG}
 yq write --inplace project-base/kubernetes/deployments/microservice-product-search.yml spec.template.spec.containers[0].image ${DOCKER_USERNAME}/microservice-product-search:${DOCKER_IMAGE_TAG}
 yq write --inplace project-base/kubernetes/deployments/microservice-product-search-export.yml spec.template.spec.containers[0].image ${DOCKER_USERNAME}/microservice-product-search-export:${DOCKER_IMAGE_TAG}
 
 # Deploy application using kubectl
 kubectl delete namespace ${JOB_NAME} || true
 kubectl create namespace ${JOB_NAME}
-kubectl apply --namespace=${JOB_NAME} --recursive -f project-base/kubernetes
+cd project-base/kubernetes/kustomize
+
+kustomize build overlays/ci | kubectl apply -f - --namespace=${JOB_NAME} --recursive
```

Comment:
```suggestion
kustomize build overlays/ci | kubectl apply -f - --namespace=${JOB_NAME}
```

The `--recursive` flag is not necessary when using data from `STDIN`, it's meant for recursive iteration over directories.